components/checkbox/flag.js

```diff
@@ -0,0 +1,18 @@
+import h from 'virtual-dom/h';
+
+import Rx from 'rx';
+
+import {labelledCheckbox} from './labelled';
+
+
+const $combine = Rx.Observable.combineLatest;
+
+export function flagCheckbox(label, initialState = false) {
```

Comment:
Seems nicer to do here than in the [`media-api` URL construction](https://github.com/guardian/grid-inspector/blob/master/media-api.js#L35-L48).